package.json

```diff
@@ -35,7 +35,8 @@
     "test:lint:js:fix": "eslint --max-warnings=0 --fix ./lib ./test ./features-legacy ./bin ./features ./doc/build-error-codes.js",
     "doc-error-codes": "node -r ts-node/register doc/build-error-codes",
     "docker:install": "docker-compose run kuzzle_node_1 npm install",
-    "docker:test:unit": "docker-compose run kuzzle_node_1 npm run test:unit"
+    "docker:test:unit": "docker-compose run kuzzle_node_1 npm run test:unit",
+    "docker:npm": "docker-compose run kuzzle_node_1 npm run --"
```

Comment:
why add this script in this PR ?

Reply:
On Mac M1, you have to run the command inside the docker, so to be easier we add this script